api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-3.0.x.xml

```diff
@@ -40,4 +40,15 @@
 		</addColumn>
 	</changeSet>
 
+	<changeSet id="TRUNK-6396-2025-08-01" author="Wikum Weerakutti">
+		<addColumn tableName="hl7_source">
```

Comment:
Now i am getting second thoughts about this. This is a table that we no longer use. It was used by the old deprecated formentry module. Does it still make much sense to add new fields for the table that we no longer use, just for the sake of compliance?

Reply:
For this table and the previous one where you created a new changeset, i would rather we remove the discriminator than create fields that we actually do not use.